packages/wallets/src/features/cashier/modules/WithdrawalLocked/WithdrawalLocked.tsx

```diff
@@ -35,29 +37,53 @@ const WithdrawalLocked: React.FC<React.PropsWithChildren> = ({ children }) => {
     const isWithdrawalLocked = status?.is_withdrawal_locked;
 
     const remainder = accountLimits?.remainder;
-    const minimumWithdrawal = activeWallet?.currency_config?.minimum_withdrawal;
+    const minimumWithdrawal = activeWallet?.currency_config?.is_crypto
+        ? activeWallet?.currency_config?.minimum_withdrawal
+        : 0.01;
     const withdrawalLimitReached = !!(
         typeof remainder !== 'undefined' &&
         typeof minimumWithdrawal !== 'undefined' &&
         +remainder < minimumWithdrawal
     );
 
+    if (isCurrencyConfigLoading) {
+        return <Loader />;
+    }
+
+    if (!isCurrencyConfigLoading && withdrawalLimitReached) {
```

Comment:
```suggestion
    if (withdrawalLimitReached) {
```
since we are checking `isCurrencyConfigLoading` above believe that here we can skip this check

Reply:
Updated